src/common.jl

```diff
@@ -1,6 +1,7 @@
 using DiffEqBase
 
 import DiffEqBase: ODEProblem, solve
+import OrdinaryDiffEq: tstop_saveat_disc_handling
 
```

Comment:
Added dependency to use the imported function. This is where Chris implements his `saveat` logic

Reply:
Nice!